Providers/Scripts/2.4x-2.5x/Scripts/Tests/run_test_OMS_nxProviders.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (c) Microsoft Corporation. All rights reserved. See license.txt for license information.
+#============================================================================
+import os
+import sys
+import glob
+
+orig_dir=os.path.realpath(os.getcwd())
+os.system('rm /tmp/omstest_cleanup* 2> /dev/null')
+os.system('rm -rf /tmp/omsext 2> /dev/null')
+
+if os.path.exists('/opt/micosoft/omsconfig'):
+    sys.stdout.write('Cannot run test_OMS_nxProviders.py when omsagent is present!') 
+    sys.exit(0)
+
+print 'Running test_OMS_nxProviders.py'
+
+# Extract the build
+os.system('touch /tmp/omstest_cleanup')
+bundle = sys.argv[1]
+bundle += '/omsagent*.universal.x64.sh' 
```

Comment:
I looked at this again - there is no need to determine the paltform, as the tests do not run any binary executables.  Only scripts and configuration files are used, and these do not change between x86 and x64.  So It's fine to use the x64 versions in all cases.